### PR TITLE
Add Vue announcer intro post

### PR DESCRIPTION
**Pull request by marcus-herrmann**
Like mentioned in vue_a11y's Discord: Modified version of https://marcus.io/blog/accessible-notifications-vue-announcer2

### CODE REVIEW

**Comment by ktquez** [comment]
@marcus-herrmann 
Add you data here https://github.com/vue-a11y/vue-a11y.com/blob/master/src/.vuepress/config/authors.js

File: src/posts/2020-10-15-vue-announcer.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: 'Using Vue Announcer'
+description: Silence isn't always golden in web apps
+summary: Vue Announcer provides an easy way to really tell what’s going on in your application to people using screen readers.
+
+categories: [aria, liveregions, screenreader,< spa, html]
```
Comment (ktquez) [inline src/posts/2020-10-15-vue-announcer.md:6]: Would you like "liveregions and screenreader" to be a category, or can they be tags?

```yaml
categories: [aria, html, spa]
tags: [liveregions, screenreader]
```

File: src/posts/2020-10-15-vue-announcer.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+- The first is dubbed "polite" and is exactly that. Just like a polite human being does not interrupt you mid-sentence in a dialogue only because they have something to say, a polite live region waits until the currently active announcements have finished. Remember: "currently active" has most of the time something to do with the position of the virtual cursor in the DOM.
+- The second one is more suitable for error messages and really important things. If your house is on fire, a polite person that waits for their turn in the dialogue, but to then yell "GET OUT! THERE'S A FIRE!" would be really inappropriate considering the emergency. This type of live region is called an "assertive" one, and it interrupts the screen reader right away.
+
+[Vue Announcer offers a way to elegantly use these live region in your Vue application](https://github.com/vue-a11y/vue-announcer).
+
+You can install it with npm (`npm install -S @vue-a11y/announcer`) or yarn (`yarn add @vue-a11y/announcer`). To include it in your app, import it and register it as a Vue plugin:
+
+```
```
Comment (ktquez) [inline src/posts/2020-10-15-vue-announcer.md:27]: Add lang in ```js

File: src/posts/2020-10-15-vue-announcer.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import Vue from 'vue'
+import VueAnnouncer from '@vue-a11y/announcer'
+
+Vue.use(VueAnnouncer)
+
+```
+
+The first step of using it is to add `<VueAnnouncer />` into your `App.vue`, for example like this:
+
+```
```
Comment (ktquez) [inline src/posts/2020-10-15-vue-announcer.md:37]: Add lang in ```html or ```vue

File: src/posts/2020-10-15-vue-announcer.md (hunk 1 above)
Comment (marcus-herrmann) [reply src/posts/2020-10-15-vue-announcer.md:6]: Can be tags

File: src/posts/2020-10-15-vue-announcer.md (hunk 2 above)
Comment (marcus-herrmann) [reply src/posts/2020-10-15-vue-announcer.md:27]: Done

File: src/posts/2020-10-15-vue-announcer.md (hunk 3 above)
Comment (marcus-herrmann) [reply src/posts/2020-10-15-vue-announcer.md:37]: Done

**Comment by ktquez** [comment]
@marcus-herrmann LGTM.

Can we publish?

**Comment by marcus-herrmann** [comment]
Sure

**Comment by marcus-herrmann** [comment]
Is the blog post held back until the Vue 3 version of announcer is out (and updated)?

**Comment by ktquez** [comment]
Hi @marcus-herrmann, how are you?

I was updating the posting page and was going to contact you for a rebase.

I hadn't thought about that, about waiting to have the post with the most updated version. What do you think?

Acho que não tem problema de publicar esse agora, qualquer novidade vamos atualizando o post.

But you decide.

I'm in vue-axe at the moment and will go to the vue-announcer probably tomorrow. I believe in the weekend be updated to version 3 and with a new approach.

---

About the current post, would it be better to decrease the size of the paragraphs a little? Maybe split into smaller blocks of text.

**Comment by ktquez** [comment]
When you update this PR, Netlify is already generating a preview.

**Comment by marcus-herrmann** [comment]
>  What do you think?

Let's wait with the merge. I'll update the post when announcer hits 3

**Comment by marcus-herrmann** [comment]
Since I find the deploy preview a little bit too hidden in the GitHub interface: https://deploy-preview-33--vue-a11y.netlify.app/posts/vue-announcer/

I also added some CSS fixes. The list display was a little off, and I tried to fix the pseudo element responsible for list bullet display.

**Comment by ktquez** [comment]
I also found it strange that the preview was so hidden.
I will see in the Netlify doc if there is a way to make it more visible.

**Comment by ktquez** [comment]
@marcus-herrmann 
Would it be possible to add the README link (doc) for v3?
https://github.com/vue-a11y/vue-announcer/blob/next/README.md

**Comment by ktquez** [comment]
**LGTM**

**Comment by marcus-herrmann** [comment]
@ktquez What an honour 😊

Updated PR.

**Pull request merged**